data_safe_haven/resources/workspace/ansible/host_vars/localhost.yaml

```
@@ -11,6 +11,10 @@ package_categories:
       - auditd
       - bats
       - clamav
+      - clamav-base
+      - clamav-daemon
+      - clamav-freshclam
```

```suggestion
      - clamav-daemon
```

These are dependencies of `clamav` and `clamav-daemon`.

(I thought when I tested this `clamav` was providing `clamd` and `clamdscan` but [this](https://packages.ubuntu.com/jammy/utils/clamav) suggests that isn't true).